examples/modular-drivers/README.md

```diff
@@ -103,33 +103,39 @@ $ cargo generate --git https://github.com/viamrobotics/micro-rdk templates/modul
 ```
 
 When prompted by the template, the project was named
-`micro-rdk-esp32-sensor-examples` and `esp32` selected for the target
+`micro-rdk-modular-driver-example` and `esp32` selected for the target
 platform. The generated project has the form of a library crate, where
 `src/lib.rs` defines an initially empty implementation of the
 well-known Micro-RDK module entry point `register_models`:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/src/lib.rs#L3-L5
+``` rust
+use micro_rdk::common::registry::{ComponentRegistry, RegistryError};
+
+pub fn register_models(_registry: &mut ComponentRegistry) -> Result<(), RegistryError>  {
+    Ok(())
+}
+```
 
 The generated project also includes a `package.metadata` section in
 its `Cargo.toml` which identifies the library crate as being a
 Micro-RDK module:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/Cargo.toml#L12-L13
+https://github.com/viamrobotics/micro-rdk/blob/fbc1783258bfefc027fd25a8cc9a1b37f6ea0524/examples/modular-drivers/Cargo.toml#L18-L19
 
 A subsequent commit adds definitions of the
-[FreeHeapSensor](https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/wifi_rssi_sensor.rs)
+[FreeHeapSensor](src/free_heap_sensor.rs)
 and
-[WifiRSSISensor](https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/free_heap_sensor.rs)
+[WifiRSSISensor](src/wifi_rssi_sensor.rs)
 
-That commit also introduces a crate-local `register_model` function
+That commit also introduces a crate-local `register_models` function
 for each sensor:
 
-- `FreeHeapSensor`: https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/free_heap_sensor.rs#L21-L25
-- `WifiRSSISensor`: https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/wifi_rssi_sensor.rs#L19-L23
+- `FreeHeapSensor`: https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/src/free_heap_sensor.rs#L23-L27
+- `WifiRSSISensor`: https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/src/wifi_rssi_sensor.rs#L23-L27
 
-Finally, the top level `register_models` entry point is updated to delegate to the `register_model` function for both sensors:
+Finally, the top level `register_models` entry point is updated to delegate to the `register_models` function for both sensors:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/9cc59d56cc35ab7cf0c471b613c5f3c2ab2ed95b/src/lib.rs#L6-L10
+https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/src/lib.rs#L10-L18
```

Comment:
The three links here should also reference fbc1783258bfefc027fd25a8cc9a1b37f6ea0524, not `main`. If possible, they should also be relative (i.e. no http://github.com/...).